cmd/object-api-input-checks.go

```diff
@@ -166,13 +166,29 @@ func checkObjectArgs(ctx context.Context, bucket, object string, obj ObjectLayer
 	if err := checkBucketExist(ctx, bucket, obj); err != nil {
 		return err
 	}
+
+	if err := objectNameLength(bucket, object); err != nil {
+		logger.LogIf(ctx, ObjectNameTooLong{
+			Bucket: bucket,
```

Comment:
No need to log since this error is a properly handled error.

Reply:
removed.